### ISSUE TITLE
jax.nn.standardize documentation still points to jax.nn.normalize

### ISSUE THREAD

**Issue by ilemhadri** [google/jax#12909]
### Description

 this PR https://github.com/google/jax/pull/9968 has deprecated `jax.nn.normalize`  in favor of `jax.nn.standardize`.

However, the online docstring still points to https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.normalize.html and there is no web link for https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.standardize.html

**Comment by mattjj** [comment]
Thanks so much for flagging this! #12912 should fix.
